### ISSUE TITLE
Rendering issue

### ISSUE THREAD

**Issue by Crypt1-C** [al132mc/chem-lib#16]
there is a rendering issue with chem lib when you have a block/nugget/ingot all of the item textures in the gui lighten up including non chemlib items

**Comment by Dark-Arcana** [comment]
Yeah it's caused by the custom rendered for the abbreviations. I've known about it for a few weeks but haven't had a chance to go back to fix it since it's pretty low priority. Good to have it documented here though.

**Comment by VeryGoodDev** [comment]
Wondering if this same rendering issue is the reason the item doesn't show up on the drawers from Functional Storage? Been using drawers heavily playing the ATM7 skyblock pack and just went to stick some cobalt dust in one and nothing shows up where it shows the item that's inside. Definitely don't wanna open another issue if it's the same problem, but wanted to at least mention it since it's also related to rendering and in case it's helpful in the process of fixing

(also mad props for such a cool mod, this is my first time playing around with ChemLib and I love it so far)

**Comment by Dark-Arcana** [comment]
Tama actually helped fix the issue already. It just hasn't been pushed and updated in ATM7Sky. Though I didn't know there was an issue in functional storage. This one was with lighting rendering funny. I'll need a new issues if you can to report that. Thanks!

**Comment by VeryGoodDev** [comment]
Ah nice, yeah I can open a new issue since it's a separate problem
